### ISSUE TITLE
[Crash report]  [Errno 2] Datei oder Verzeic

### ISSUE THREAD

**Issue by gilunixuser** [bottlesdevs/Bottles#1547]
This crash report was generated by Bottles.

**Details**
* **package**: Flatpak
* **version**: 2022.5.14-trento-3

**Log**
```python3
[Errno 2] Datei oder Verzeichnis nicht gefunden: '/run/user/1000/doc/57f8016b/Steuerfälle'  File "/app/share/bottles/bottles/utils/threading.py", line 64, in __target
    result = self.task_func(*args, **kwargs)

  File "/app/share/bottles/bottles/backend/wine/winedbg.py", line 80, in wait_for_process
    processes = self.get_processes()

  File "/app/share/bottles/bottles/backend/wine/winedbg.py", line 30, in get_processes
    res = self.launch(

  File "/app/share/bottles/bottles/backend/wine/wineprogram.py", line 58, in launch
    res = WineCommand(

  File "/app/share/bottles/bottles/backend/wine/winecommand.py", line 452, in run
    proc = subprocess.Popen(

  File "/usr/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,

  File "/usr/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)

```

**Comment by mirkobrombin** [comment]
`/run/user/1000/doc/57f8016b/Steuerfälle` ensure bottles has access to the path https://docs.usebottles.com/flatpak/expose-directories

**Comment by gilunixuser** [comment]
"Reported at least 30 times and we can't fix, move files to sandbox or give Bottles access to path containing files."
I used a tax return software. When I was in the process  of sending the tax return declaration to the tax office the software respectively bottles crashed. There is nothing obvious that the software needed to access to another path.
So the given solution does not  help.

**Comment by mirkobrombin** [comment]
The setting may be lost because the path is unmounted. The solution is correct, you have to allow the flatpak to access that path permanently. Try giving it access to the entire media folder.

**Comment by gilunixuser** [comment]
1. "The setting may be lost because the path is unmounted"
How can I mount this path? There is no "Ok" or "Accept" button in flatseal neither. I simply entered the path I took from my Linux system file manager. Is there another logic in flatseal?
2. Secondly my tax return software on bottles only wanted to send my tax return declaration to the tax office and bottles crashed than. I do not understand why it would need special file access which was not needed during all of use before??
It is not that obvious and easy to solve this problem as far I can see it now.
